monday/client.py

```diff
@@ -7,23 +7,34 @@
 """
 
 from .__version__ import __version__
-from .resources import ItemResource, UpdateResource, TagResource, BoardResource, UserResource, GroupResource, ComplexityResource, WorkspaceResource, NotificationResource
+from .resources import CustomResource, ItemResource, ColumnsResource, UpdateResource, TagResource, BoardResource, \
+    UserResource, GroupResource, ComplexityResource, WorkspaceResource, NotificationResource, MeResource
+
+_DEFAULT_HEADERS = {}
 
 
 class MondayClient:
-    def __init__(self, token):
+    def __init__(self, token, headers=None):
         """
-        :param token: API Token for the new :class:`BaseResource` object.
+        :param token: API token for the new :class:`BaseResource` object.
+        :param headers: (optional) headers for the new :class:`BaseResource` object.
         """
-        self.items = ItemResource(token=token)
-        self.updates = UpdateResource(token=token)
-        self.tags = TagResource(token=token)
-        self.boards = BoardResource(token=token)
-        self.users = UserResource(token=token)
-        self.groups = GroupResource(token=token)
-        self.complexity = ComplexityResource(token=token)
-        self.workspaces = WorkspaceResource(token=token)
-        self.notifications = NotificationResource(token=token)
+
+        if not headers:
+            headers = _DEFAULT_HEADERS.copy()
+
+        self.custom = CustomResource(token=token, headers=headers)
+        self.items = ItemResource(token=token, headers=headers)
+        self.columns = ColumnsResource(token=token, headers=headers)
+        self.updates = UpdateResource(token=token, headers=headers)
+        self.tags = TagResource(token=token, headers=headers)
+        self.boards = BoardResource(token=token, headers=headers)
+        self.users = UserResource(token=token, headers=headers)
+        self.groups = GroupResource(token=token, headers=headers)
+        self.complexity = ComplexityResource(token=token, headers=headers)
+        self.workspaces = WorkspaceResource(token=token, headers=headers)
+        self.notifications = NotificationResource(token=token, headers=headers)
+        self.me = MeResource(token=token, headers=headers)
 
     def __str__(self):
         return f'MondayClient {__version__}'
```

Comment:
Worth having the API Version as part of the string rep of the client maybe?

Reply:
Probably, but I'm not sure if it should be the default API version or the one actually used by that specific client.  I think specific client makes more sense, but I could see an argument for the other way.  Either way, I'm skipping this for the moment, as it's not exactly essential.